### DISCUSSION TITLE
Can't save this page - why?

### DISCUSSION THREAD

**Discussion by tchflws** [gildas-lormeau/SingleFile#687] [General]
So, I was trying to download

https://www.zeit.de/digital/2021-05/uploadfilter-plattformen-algorithmen-urheberrechtsreform-bundestag/komplettansicht

for later reference but when I click on single file, I'm moved to the main url. Maybe you can have a look? Thx!

**Comment by gildas-lormeau** [comment]
Make sure you don't scroll too much before saving the page.
